index.html

```diff
@@ -72,43 +67,22 @@
             <div class="container custom-container">
                 <div class="main-header-bg">
                     <div class="row align-items-center justify-content-between">
-                        <div class="col-xl-3 col-lg-3 col-md-6 col-6">
+                        <div class="col-xl-2 col-lg-3 col-md-6 col-6">
                             <div class="logo">
-                                <a class="logo-img" href="index.html"><img src="assets/img/logo/header_logo_one.png" alt=""></a>
```

Comment:
Let's Allow this on the page, I have their logo and I'm already working  on their logos, so lets allow this, revert this back to the default

Reply:
Okay, no wahala